docs/en/examples/using_elementalArea_with_dataObject.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Using ElementalArea with DataObject
+summary: Learn how to add ElementalArea to your models and manage it in ModalAdmin
+---
+
+# Using ElementalArea with DataObject
+
+## Creaating Model and adding ElementalArea
+
+In this example the class BlogPost will be created, it will extend DataObject class. Also new BlogPost Admin section will be created in Admin panel where CMS user can manage the BlogPosts. 
+
+Let's look at an example:
+
+**app/src/Admins/BlogPostsAdmin.php**
+
+```php
+namespace App\Admins;
+
+use App\Models\BlogPost;
+use SilverStripe\Admin\ModelAdmin;
+
+class BlogPostsAdmin extends ModelAdmin
+{
+
+    private static string $url_segment = 'blog-posts-admin';
+
+    private static string $menu_title = 'Blog Posts';
+
+    private static string $menu_icon_class = 'font-icon-block-banner';
+
+    private static array $managed_models = [
+        BlogPost::class,
+    ];
+}
+```
+
+``` $managed_models ``` - this static variable connects BlogPostAdmin with BlogPost, which will be created in the following code.
```

Comment:
```suggestion
`$managed_models` - this static variable connects BlogPostAdmin with BlogPost, which will be created in the following code.
```
Three backticks usually denote a block of code rather than an inline code reference.